src/modules/site-v2/templates/order/order.html

```diff
@@ -57,11 +57,11 @@
                 {% for item in cartItems %}
                 <tr class='cartItem'>
                     <td>{{ loop.index }}</td>
-                    <td><em>{{ item.species }}</em></td>
+                    <td><em>{{ item.species|capitalize|replace('_', '. ') }}</em></td>

```

Comment:
It's a bit hidden, but `item.species.short_name` should do this same operation (see line ~56 in `models/datastore/species.py`).  I don't think we'll ever need to change how the "short name" works, but if we did, it's good to have it defined in just one place.